### ISSUE TITLE
Connecting to WiFi causes error on Raspberry Pi 4

### ISSUE THREAD

**Issue by IgorDancik** [home-assistant/supervisor#5352]
### Describe the issue you are experiencing

when trying to connect to wifi from HA:
- all available AP visible
- selecting any AP + option (automatic, static) 
- produces mesage: home assistant connect wifi Object does not exist at path “/org/freedesktop/NetworkManager/ActiveConnection/6”

### What operating system image do you use?

rpi4-64  (Raspberry Pi 4/400 64-bit OS)

### What version of Home Assistant Operating System is installed?

13.1

### Did the problem occur after upgrading the Operating System?

No

### Hardware details

RPi 4
Core2024.10.1
Supervisor2024.10.0
Operating System13.1
Frontend20241002.2

### Steps to reproduce the issue

1. install new HA on LAN
2. select AP + connect options 
3. HP refuses to connect + produces mesage: home assistant connect wifi Object does not exist at path “/org/freedesktop/NetworkManager/ActiveConnection/6”
I manage to connect to wifi in previous versions under similar conditions...

### Anything in the Supervisor logs that might be useful for us?

```txt
2024-10-09 05:07:13.478 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
2024-10-09 05:07:13.478 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state running
2024-10-09 05:07:13.478 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
2024-10-09 05:10:20.142 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
2024-10-09 05:10:21.423 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-10-09 05:23:38.375 INFO (MainThread) [supervisor.host.info] Updating local host information
2024-10-09 05:23:39.004 INFO (MainThread) [supervisor.host.services] Updating service information
2024-10-09 05:23:39.018 INFO (MainThread) [supervisor.host.network] Updating local network information
2024-10-09 05:23:39.225 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
2024-10-09 05:23:39.236 INFO (MainThread) [supervisor.host.manager] Host information reload completed
2024-10-09 05:40:21.737 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-10-09 06:07:13.498 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state running
2024-10-09 06:07:13.499 INFO (MainThread) [supervisor.resolution.checks.base] Run check for docker_config/system
2024-10-09 06:07:13.499 INFO (MainThread) [supervisor.resolution.checks.base] Run check for disabled_data_disk/system
2024-10-09 06:07:13.500 INFO (MainThread) [supervisor.resolution.checks.base] Run check for free_space/system
2024-10-09 06:07:13.504 INFO (MainThread) [supervisor.resolution.checks.base] Run check for multiple_data_disks/system
2024-10-09 06:07:13.505 INFO (MainThread) [supervisor.resolution.checks.base] Run check for pwned/addon
2024-10-09 06:07:13.665 INFO (MainThread) [supervisor.resolution.checks.base] Run check for trust/supervisor
2024-10-09 06:07:13.679 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_failed/dns_server
2024-10-09 06:07:13.679 INFO (MainThread) [supervisor.resolution.checks.base] Run check for ipv4_connection_problem/system
2024-10-09 06:07:13.680 INFO (MainThread) [supervisor.resolution.checks.base] Run check for security/core
2024-10-09 06:07:13.681 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_ipv6_error/dns_server
2024-10-09 06:07:13.681 INFO (MainThread) [supervisor.resolution.check] System checks complete
2024-10-09 06:07:13.681 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state running
2024-10-09 06:07:13.845 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
2024-10-09 06:07:13.845 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state running
2024-10-09 06:07:13.845 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
2024-10-09 06:10:22.060 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-10-09 06:40:22.375 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-10-09 07:07:13.883 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state running
2024-10-09 07:07:13.884 INFO (MainThread) [supervisor.resolution.checks.base] Run check for docker_config/system
2024-10-09 07:07:13.885 INFO (MainThread) [supervisor.resolution.checks.base] Run check for disabled_data_disk/system
2024-10-09 07:07:13.885 INFO (MainThread) [supervisor.resolution.checks.base] Run check for free_space/system
2024-10-09 07:07:13.886 INFO (MainThread) [supervisor.resolution.checks.base] Run check for multiple_data_disks/system
2024-10-09 07:07:13.888 INFO (MainThread) [supervisor.resolution.checks.base] Run check for pwned/addon
2024-10-09 07:07:14.051 INFO (MainThread) [supervisor.resolution.checks.base] Run check for trust/supervisor
2024-10-09 07:07:14.070 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_failed/dns_server
2024-10-09 07:07:14.071 INFO (MainThread) [supervisor.resolution.checks.base] Run check for ipv4_connection_problem/system
2024-10-09 07:07:14.071 INFO (MainThread) [supervisor.resolution.checks.base] Run check for security/core
2024-10-09 07:07:14.072 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_ipv6_error/dns_server
2024-10-09 07:07:14.072 INFO (MainThread) [supervisor.resolution.check] System checks complete
2024-10-09 07:07:14.073 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state running
2024-10-09 07:07:14.239 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
2024-10-09 07:07:14.240 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state running
2024-10-09 07:07:14.240 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
2024-10-09 07:10:20.355 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
2024-10-09 07:10:22.283 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/music-assistant/home-assistant-addon repository
2024-10-09 07:10:22.288 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/esphome/home-assistant-addon repository
2024-10-09 07:10:22.295 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
2024-10-09 07:10:22.299 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
2024-10-09 07:10:22.688 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-10-09 07:10:24.545 INFO (MainThread) [supervisor.store] Loading add-ons from store: 80 all - 0 new - 0 remove
2024-10-09 07:10:24.545 INFO (MainThread) [supervisor.store] Loading add-ons from store: 80 all - 0 new - 0 remove
2024-10-09 07:30:19.253 INFO (MainThread) [supervisor.host.info] Updating local host information
2024-10-09 07:30:19.865 INFO (MainThread) [supervisor.host.services] Updating service information
2024-10-09 07:30:19.879 INFO (MainThread) [supervisor.host.network] Updating local network information
2024-10-09 07:30:20.085 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
2024-10-09 07:30:20.097 INFO (MainThread) [supervisor.host.manager] Host information reload completed
2024-10-09 07:40:23.010 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-10-09 08:07:14.279 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state running
2024-10-09 08:07:14.280 INFO (MainThread) [supervisor.resolution.checks.base] Run check for docker_config/system
2024-10-09 08:07:14.281 INFO (MainThread) [supervisor.resolution.checks.base] Run check for disabled_data_disk/system
2024-10-09 08:07:14.281 INFO (MainThread) [supervisor.resolution.checks.base] Run check for free_space/system
2024-10-09 08:07:14.285 INFO (MainThread) [supervisor.resolution.checks.base] Run check for multiple_data_disks/system
2024-10-09 08:07:14.286 INFO (MainThread) [supervisor.resolution.checks.base] Run check for pwned/addon
2024-10-09 08:07:14.453 INFO (MainThread) [supervisor.resolution.checks.base] Run check for trust/supervisor
2024-10-09 08:07:14.473 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_failed/dns_server
2024-10-09 08:07:14.474 INFO (MainThread) [supervisor.resolution.checks.base] Run check for ipv4_connection_problem/system
2024-10-09 08:07:14.474 INFO (MainThread) [supervisor.resolution.checks.base] Run check for security/core
2024-10-09 08:07:14.475 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_ipv6_error/dns_server
2024-10-09 08:07:14.475 INFO (MainThread) [supervisor.resolution.check] System checks complete
2024-10-09 08:07:14.475 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state running
2024-10-09 08:07:14.642 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
2024-10-09 08:07:14.643 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state running
2024-10-09 08:07:14.643 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
2024-10-09 08:10:23.321 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-10-09 08:40:23.635 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-10-09 09:07:14.667 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state running
2024-10-09 09:07:14.667 INFO (MainThread) [supervisor.resolution.checks.base] Run check for docker_config/system
2024-10-09 09:07:14.668 INFO (MainThread) [supervisor.resolution.checks.base] Run check for disabled_data_disk/system
2024-10-09 09:07:14.669 INFO (MainThread) [supervisor.resolution.checks.base] Run check for free_space/system
2024-10-09 09:07:14.672 INFO (MainThread) [supervisor.resolution.checks.base] Run check for multiple_data_disks/system
2024-10-09 09:07:14.674 INFO (MainThread) [supervisor.resolution.checks.base] Run check for pwned/addon
2024-10-09 09:07:14.836 INFO (MainThread) [supervisor.resolution.checks.base] Run check for trust/supervisor
2024-10-09 09:07:14.854 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_failed/dns_server
2024-10-09 09:07:14.855 INFO (MainThread) [supervisor.resolution.checks.base] Run check for ipv4_connection_problem/system
2024-10-09 09:07:14.855 INFO (MainThread) [supervisor.resolution.checks.base] Run check for security/core
2024-10-09 09:07:14.856 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_ipv6_error/dns_server
2024-10-09 09:07:14.857 INFO (MainThread) [supervisor.resolution.check] System checks complete
2024-10-09 09:07:14.857 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state running
2024-10-09 09:07:15.022 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
2024-10-09 09:07:15.023 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state running
2024-10-09 09:07:15.023 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
2024-10-09 09:10:22.272 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
2024-10-09 09:10:23.956 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
2024-10-09 09:37:00.101 INFO (MainThread) [supervisor.host.info] Updating local host information
2024-10-09 09:37:00.724 INFO (MainThread) [supervisor.host.services] Updating service information
2024-10-09 09:37:00.737 INFO (MainThread) [supervisor.host.network] Updating local network information
2024-10-09 09:37:00.951 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
2024-10-09 09:37:00.961 INFO (MainThread) [supervisor.host.manager] Host information reload completed
2024-10-09 09:40:24.280 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
```

### Anything in the Host logs that might be useful for us?

```txt
2024-10-08 23:10:17.397 NewHA kernel: audit: type=1334 audit(1728429017.391:219): prog-id=56 op=LOAD
2024-10-08 23:10:17.397 NewHA kernel: audit: type=1334 audit(1728429017.391:220): prog-id=57 op=LOAD
2024-10-08 23:10:17.400 NewHA kernel: audit: type=1334 audit(1728429017.395:221): prog-id=58 op=LOAD
2024-10-08 23:10:17.426 NewHA systemd[1]: Starting Hostname Service...
2024-10-08 23:10:17.748 NewHA systemd[1]: Started Hostname Service.
2024-10-08 23:10:17.764 NewHA kernel: audit: type=1334 audit(1728429017.759:222): prog-id=59 op=LOAD
2024-10-08 23:10:17.768 NewHA kernel: audit: type=1334 audit(1728429017.763:223): prog-id=60 op=LOAD
2024-10-08 23:10:17.769 NewHA kernel: audit: type=1334 audit(1728429017.763:224): prog-id=61 op=LOAD
2024-10-08 23:10:17.784 NewHA systemd[1]: Starting Time & Date Service...
2024-10-08 23:10:18.048 NewHA systemd[1]: Started Time & Date Service.
2024-10-08 23:10:47.787 NewHA systemd[1]: systemd-hostnamed.service: Deactivated successfully.
2024-10-08 23:10:49.332 NewHA kernel: audit: type=1334 audit(1728429047.879:225): prog-id=58 op=UNLOAD
2024-10-08 23:10:49.332 NewHA kernel: audit: type=1334 audit(1728429047.879:226): prog-id=57 op=UNLOAD
2024-10-08 23:10:49.332 NewHA kernel: audit: type=1334 audit(1728429047.879:227): prog-id=56 op=UNLOAD
2024-10-08 23:10:49.333 NewHA systemd[1]: systemd-timedated.service: Deactivated successfully.
2024-10-08 23:10:49.336 NewHA kernel: audit: type=1334 audit(1728429049.331:228): prog-id=61 op=UNLOAD
2024-10-08 23:10:49.337 NewHA kernel: audit: type=1334 audit(1728429049.331:229): prog-id=60 op=UNLOAD
2024-10-08 23:10:49.337 NewHA kernel: audit: type=1334 audit(1728429049.331:230): prog-id=59 op=UNLOAD
2024-10-08 23:12:27.421 NewHA systemd-resolved[505]: Grace period over, resuming full feature set (UDP+EDNS0) for DNS server 192.168.0.1.
2024-10-08 23:12:27.456 NewHA systemd-resolved[505]: Grace period over, resuming full feature set (UDP+EDNS0) for DNS server 192.168.0.1.
2024-10-08 23:12:27.456 NewHA systemd-resolved[505]: Using degraded feature set UDP instead of UDP+EDNS0 for DNS server 192.168.0.1.
2024-10-08 23:12:27.456 NewHA systemd-resolved[505]: Using degraded feature set UDP instead of UDP+EDNS0 for DNS server 192.168.0.1.
2024-10-08 23:17:01.062 NewHA systemd[1]: Starting Cleanup of Temporary Directories...
2024-10-08 23:17:01.101 NewHA systemd[1]: systemd-tmpfiles-clean.service: Deactivated successfully.
2024-10-08 23:17:01.102 NewHA systemd[1]: Finished Cleanup of Temporary Directories.
2024-10-09 01:16:57.404 NewHA kernel: audit: type=1334 audit(1728436617.399:231): prog-id=62 op=LOAD
2024-10-09 01:16:57.405 NewHA kernel: audit: type=1334 audit(1728436617.399:232): prog-id=63 op=LOAD
2024-10-09 01:16:57.405 NewHA kernel: audit: type=1334 audit(1728436617.399:233): prog-id=64 op=LOAD
2024-10-09 01:16:57.435 NewHA systemd[1]: Starting Hostname Service...
2024-10-09 01:16:57.704 NewHA systemd[1]: Started Hostname Service.
2024-10-09 01:16:57.718 NewHA kernel: audit: type=1334 audit(1728436617.715:234): prog-id=65 op=LOAD
2024-10-09 01:16:57.718 NewHA kernel: audit: type=1334 audit(1728436617.715:235): prog-id=66 op=LOAD
2024-10-09 01:16:57.719 NewHA kernel: audit: type=1334 audit(1728436617.715:236): prog-id=67 op=LOAD
2024-10-09 01:16:57.744 NewHA systemd[1]: Starting Time & Date Service...
2024-10-09 01:16:57.978 NewHA systemd[1]: Started Time & Date Service.
2024-10-09 01:17:27.744 NewHA systemd[1]: systemd-hostnamed.service: Deactivated successfully.
2024-10-09 01:17:29.472 NewHA kernel: audit: type=1334 audit(1728436647.846:237): prog-id=64 op=UNLOAD
2024-10-09 01:17:29.472 NewHA kernel: audit: type=1334 audit(1728436647.846:238): prog-id=63 op=UNLOAD
2024-10-09 01:17:29.472 NewHA kernel: audit: type=1334 audit(1728436647.846:239): prog-id=62 op=UNLOAD
2024-10-09 01:17:29.473 NewHA systemd[1]: systemd-timedated.service: Deactivated successfully.
2024-10-09 01:17:29.476 NewHA kernel: audit: type=1334 audit(1728436649.470:240): prog-id=67 op=UNLOAD
2024-10-09 01:17:29.477 NewHA kernel: audit: type=1334 audit(1728436649.470:241): prog-id=66 op=UNLOAD
2024-10-09 01:17:29.477 NewHA kernel: audit: type=1334 audit(1728436649.470:242): prog-id=65 op=UNLOAD
2024-10-09 03:23:38.396 NewHA kernel: audit: type=1334 audit(1728444218.390:243): prog-id=68 op=LOAD
2024-10-09 03:23:38.397 NewHA kernel: audit: type=1334 audit(1728444218.390:244): prog-id=69 op=LOAD
2024-10-09 03:23:38.400 NewHA kernel: audit: type=1334 audit(1728444218.394:245): prog-id=70 op=LOAD
2024-10-09 03:23:38.426 NewHA systemd[1]: Starting Hostname Service...
2024-10-09 03:23:38.691 NewHA systemd[1]: Started Hostname Service.
```

### System information

-

**Comment by agners** [comment]
> This might be fallout of #5293 😢

Reverting this PR does not seem to fix the issue.

Using the CLI seems to work, so this is likely not a Supervisor issue. You can set the network configuration via HA CLI as follows:

```
ha network info
ha network update <wifi-interface> --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid <ssid> --wifi-psk <pw>
```

**Comment by agners** [comment]
Confirmed that this is a frontend issue introduced in 2024.10.0 with https://github.com/home-assistant/frontend/pull/22121. This will be resolved with https://github.com/home-assistant/frontend/pull/22377, should go into 2024.10.3 and 2024.11.0. Thanks for your report.

**Comment by IgorDancik** [comment]
It works - perfect !    
:-)

**Comment by Ivan70Zsivi** [comment]
I am a newby....please could you tell me how to overcome this problem? I have the same issue

**Comment by SkyFocusNL** [comment]
I have upgraded my hardware yesterday from rb4 to rb5 using a full-backup of my HAOS setup. 

The old rb4 had no problems.

The rb5 only works on LAN. I have the same error when trying to enable Wifi through the GUI.
I tried the CLI and this also gives me an error:
"~ Error: Activating connection failed, check connection settings."

I've checked and the settings  are ok ...

**Comment by IgorDancik** [comment]
This is the solution (delivered by agners) which worked for me:

> > This might be fallout of #5293 😢
> 
> Reverting this PR does not seem to fix the issue.
> 
> Using the CLI seems to work, so this is likely not a Supervisor issue. You can set the network configuration via HA CLI as follows:
> 
> ```
> ha network info
> ha network update <wifi-interface> --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid <ssid> --wifi-psk <pw>
> ```

In detail:

GO to terminal / cli
GET: ha network info
TYPE:  ha network update <wifi-interface> --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid <ssid> --wifi-psk <pw>
.... with your <data> from ha-> system->network
HA restores network manager
REBOOT ha - to make wlan0 available and from ha-> system->network setup connection
REBOOT ha again

Rem:  reboots are necessary to activate changes

**Comment by ejencik** [comment]
you need to specify interface wlan0 in command like:
ha network update wlan0 --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid --wifi-psk
then it works perfectly

many thanks for solving this issue !!!

**Comment by GiordanoMatheus** [comment]
Fiz um downgrade para a versão 2024.9.1, após o downgrade configurei o WI-FI.
Após tudo funcionar, fiz o update para a versão 2024.10.3.

Está tudo funcionando normalmente agora.

**Comment by seriousManual** [comment]
> This will be resolved with https://github.com/home-assistant/frontend/pull/22377, should go into 2024.10.3 and 2024.11.0.

I upgraded to 2024.10.3 and have still the same issue.

I've just started HA and I'm not too familiar, so I have a question regarding the fix with the `ha ...` command the was mentioned.
How do I get to a shell where I can actually use the `ha` command?

**Comment by IgorDancik** [comment]
implement terminal add-on ad go to terminal GUI

**Comment by rp2019433** [comment]
> > This will be resolved with [home-assistant/frontend#22377](https://github.com/home-assistant/frontend/pull/22377), should go into 2024.10.3 and 2024.11.0.
> 
> I upgraded to 2024.10.3 and have still the same issue.
> 
> I've just started HA and I'm not too familiar, so I have a question regarding the fix with the `ha ...` command the was mentioned. How do I get to a shell where I can actually use the `ha` command?

Same here. I still have the problem even after upgrading to  2024.10.3.

**Comment by ferdyxd** [comment]
> > > This will be resolved with [home-assistant/frontend#22377](https://github.com/home-assistant/frontend/pull/22377), should go into 2024.10.3 and 2024.11.0.
> > 
> > 
> > I upgraded to 2024.10.3 and have still the same issue.
> > I've just started HA and I'm not too familiar, so I have a question regarding the fix with the `ha ...` command the was mentioned. How do I get to a shell where I can actually use the `ha` command?
> 
> Same here. I still have the problem even after upgrading to 2024.10.3.

Me too, same problem with the updating core 2024.10.3.

I've tried the command ha network update wlan0 --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid --wifi-psk in the CLI with the addon "advanced SSH & Web Terminal" and the terminal answered me with the same error than the HA gave me trying to configurared in the frontend network option. " Object does not exist at path "org/.......

**Comment by ferdyxd** [comment]
> > > > This will be resolved with [home-assistant/frontend#22377](https://github.com/home-assistant/frontend/pull/22377), should go into 2024.10.3 and 2024.11.0.
> > > 
> > > 
> > > I upgraded to 2024.10.3 and have still the same issue.
> > > I've just started HA and I'm not too familiar, so I have a question regarding the fix with the `ha ...` command the was mentioned. How do I get to a shell where I can actually use the `ha` command?
> > 
> > 
> > Same here. I still have the problem even after upgrading to 2024.10.3.
> 
> Me too, same problem with the updating core 2024.10.3.
> 
> I've tried the command ha network update wlan0 --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid --wifi-psk in the CLI with the addon "advanced SSH & Web Terminal" and the terminal answered me with the same error than the HA gave me trying to configurared in the frontend network option. " Object does not exist at path "org/.......

adding more information just to someone understand what is happeping...
:  
This is de debug with the command ha network update wlan0 --ipv4-method auto --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid [myssid]--wifi-psk [mypass] --log-level debug


DEBU[0000] Adding homedir to searchpath                  homedir=/home/hassio
INFO[0000] No configfile found
DEBU[0000] Debug flags                                   apiToken=e1f68ed5fb6fbf44911011cb2e1baba09897d8ed405a1485374b35906a664bd cfgFile= endpoint=supervisor logLevel=debug noProgress=false rawJSON=false
DEBU[0000] network update                                args="[wlan0]"
DEBU[0000] [GenerateURI]                                 base=supervisor command="interface/{interface}/update" section=network
DEBU[0000] [GenerateURI] Result                          uri="http://supervisor/network/interface/{interface}/update" url="http://supervisor/network/interface/%7Binterface%7D/update" url(string)="http://supervisor/network/interface/{interface}/update"
DEBU[0000] Request body                                  options="map[enabled:true ipv4:map[method:auto] wifi:map[auth:wpa-psk mode:infrastructure psk: [mypass] ssid: [myssid]]]"
DEBU[0025] Response                                      body="{\"result\":\"error\",\"message\":\"Activating connection failed, check connection settings.\"}" fields.time=25.564251185s headers="map[Content-Length:[87] Content-Type:[application/json; charset=utf-8] Date:[Tue, 22 Oct 2024 13:32:37 GMT] Server:[Python/3.12 aiohttp/3.10.10]]" received-at="2024-10-22 15:32:37.496595542 +0200 CEST m=+25.572606602" request="&{POST http://supervisor/network/interface/wlan0/update HTTP/1.1 1 1 map[Accept:[application/json] Authorization:[Bearer 06a664bd] 

Content-Type:[application/json] User-Agent:[go-resty/2.13.1 (https://github.com/go-resty/resty)]] 0x40001ee870 0x2a9c30 142 [] false supervisor map[] map[] <nil> map[]   <nil> <nil> <nil> 0x40000d0040}" status="400 Bad Request" statuscode=400
Error: Activating connection failed, check connection settings.

**Comment by seriousManual** [comment]
> implement terminal add-on ad go to terminal GUI

weird, i must have missed that, thanks.
in my case activating wifi via the `ha ...` command worked.

**Comment by andriuskr** [comment]
For me even through CLI does not work. error is Object does not exist at path “/org/freedesktop/NetworkManager/ActiveConnection/25”

**Comment by olivier8964** [comment]
> I was having the same issue when trying to enable wifi through the web interface. CLI works perfectly. I had to use it a bit different:
> 
> ha network update wlan0 --wifi-auth “wpa-psk” --wifi-ssid “your-network” --wifi-psk “your-password” --ipv4-method “auto”

Thanks, it works

**Comment by andriuskr** [comment]
<img width="1091" alt="image" src="https://github.com/user-attachments/assets/65a813d7-a20e-4ca8-9f5c-41d000050e6d">

**Comment by cengiz28** [comment]
> <img alt="Bild" width="1091" src="https://private-user-images.githubusercontent.com/13429780/379552615-65a813d7-a20e-4ca8-9f5c-41d000050e6d.png? jwt=eyJhbGciOiJIUzI1NiInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIcocmF3LmdpdGh1YnVjb1YnVzZXJb250LmNvbSIImtleSI6ImtleTUiLCJleHAiOjOjE3Mjk3Mjk3NTY4MzksIm5i5iOTcyOtc1NjUzOtozGF0aCI6i8MzQyOTc4MC8zNTI2MTUtNjVhODEzZDctYTIwZS00Y2E4LTlmNWMtNDFkMDAwMDUwZuTZnBuZz9YLUFtei1Bb  


ha Netzwerk-Update wlan0 --wifi-auth wpa-psk--wifi-ssid your-network --wifi-psk your-password --ipv4-method auto

**Comment by Plermpel** [comment]
Let me jump in here.
I had the same issue and wasn't even able to connect to my wifi via command because me password has a "?" in it.
I had to escape the "?" with a "\" before it, to tread it as a normal character in the terminal, like that:

ha network update wlan0 --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid MYSSID --wifi-psk pass\?word

Maybe that happens with other secial characters too, hope I helped someone.

**Comment by fabio0863** [comment]
Good morning, I also had the same problem on wlan0. I use wlan0 to connect to a huawei solar inverter that can only be reached via wifi. I had temporarily solved the bug by downgrading the core to the 2024.09 release. After reading this discussion I tried to update the 2024.10.3 core and launched the command -> **ha network update wlan0 --wifi-auth wpa-psk --wifi-ssid SUN2000-BT22C0**xxxxxx **--wifi-psk Chang**xxx **--ipv4-method auto**
I must say that wlan0 immediately connected, obtaining an IP address (from the huawei network 192.168. 200.0).  Unfortunately, after a few minutes I noticed that the connection was unstable and continued to drop. So I reloaded my 2024.09 backup and continue to hope for a new release that solves this problem. Thanks to everyone.

**Comment by ferdyxd** [comment]
My problem is that I changed the USB SSD from the top of the Raspberry Pi to the bottom of the Pi, and apparently that is what caused HA to not be able to connect to the Huawei WiFi. The Wifi was already connected as soon as I changed the USB port to the one on top of the PI and reset it.

**Comment by AngryUndeadDad** [comment]
I still can't connect using any of those methods. I get "Error: Activating connection failed" in the terminal. I've double checked the password and everything.

**Comment by SkyFocusNL** [comment]
> Has anyone tried to see if the problem has been fixed with version 2024.11.1 ?

I just tried the new 2024.11.1 release and it fixed it for me on a RP 5 with HAOS ...
